code/modules/antagonists/changeling/powers/transform.dm

```diff
@@ -139,6 +139,21 @@
 		return
 	. = ..()
 
+/obj/item/changeling/id
+	/// Cached flat icon of the ID
+	var/icon/cached_flat_icon
+
+/**
+ * Returns cached flat icon of the ID, creates one if there is not one already cached
+ */
+/obj/item/changeling/id/proc/get_cached_flat_icon()
+	if(!cached_flat_icon)
+		cached_flat_icon = getFlatIcon(src)
+	return cached_flat_icon
+
+/obj/item/changeling/id/get_examine_string(mob/user, thats = FALSE)
```

Comment:
What's the point of this, this is only visible to the changeling who knows it's fake ? Duplicating ID code (that includes getFlatIcon on top of that ) seems to be pretty pointless here.

Reply:
This is visible to everyone, see the attached issue. Everyone can examine the changeling and find out ID is fake, because it was always missing overlays.

Reply:
Ah i mixed this up with the show ID option.